<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>main</title>
  <style>
    html {
      line-height: 1.5;
      font-family: Georgia, serif;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 36em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      word-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
        font-size: 12pt;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin: 1em 0;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.4em;
    }
    h5, h6 {
      font-size: 1em;
      font-style: italic;
    }
    h6 {
      font-weight: normal;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1em 0 1em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      color: #606060;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      font-size: 85%;
      margin: 0;
    }
    pre {
      margin: 1em 0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
    }
    .sourceCode {
     background-color: transparent;
     overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin: 1em 0;
    }
    table {
      margin: 1em 0;
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
      font-variant-numeric: lining-nums tabular-nums;
    }
    table caption {
      margin-bottom: 0.75em;
    }
    tbody {
      margin-top: 0.5em;
      border-top: 1px solid #1a1a1a;
      border-bottom: 1px solid #1a1a1a;
    }
    th {
      border-top: 1px solid #1a1a1a;
      padding: 0.25em 0.5em 0.25em 0.5em;
    }
    td {
      padding: 0.125em 0.5em 0.25em 0.5em;
    }
    header {
      margin-bottom: 4em;
      text-align: center;
    }
    #TOC li {
      list-style: none;
    }
    #TOC a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<div class="LARGE">
<p><strong>Synthetic Biology Tools to Investigate Redox Signal Transduction in Photosynthetic Bacteria</strong></p>
</div>
<p>Alberto Scarampi<span class="math inline"><sup>1</sup></span>, Christopher J. Howe<span class="math inline"><sup>1</sup></span> <span class="math inline"><sup>1</sup></span>Hopkins Building, Department of Biochemistry, University of Cambridge, UK<br />
</p>
<hr />
<h1 class="unnumbered" id="summary">Summary</h1>
<p>Many species of bacteria, similarly to eukaryotic cells, possess genetically-encoded pathways implicated in sensing and generating extracellular electrical signals. The study of such biochemical mechanisms in model prokaryotes (e.g. <em>Escherichia coli</em>, <em>Bacillus subtilis</em>) provided interesting insights on how these “simple" cells coordinate complex collective behaviours using redox cues and has been exploited to exchange information between electrical and biological devices by genetic means.</p>
<p>However, a mechanistic understanding on the genetic regulation of redox signal transduction is only available for a few heterotrophic bacteria. In addition synthetic genetic reporter devices that can be used to probe the genetic expression from redox-sensitive promoters can only - to date - operate in anaerobic conditions. This raises the question of how these bacteria can utilise oxygen-sensitive redox signal transduction pathways while living in aerobic environments.</p>
<p>This project aims to address this question by investigating redox signal transduction pathways in oxygen-evolving photosynthetic bacteria, which comprise a class of bacteria often probed bioelectrochemically, but whose electrogenetic landscape remains uncharacterised. This will be addressed experimentally by developing a framework to characterise a specific genetic circuit (the prqRA operon), whose expression is significantly upregulated in conditions promoting oxidative stress. The main hypothesis that underlies this project is that when bacteria are employed in bioelectrochemical systems they may experience both direct and indirect oxidative stresses, resulting in upregulation of stress-quenching adaptive responses.</p>
<p>Having evolved the ability to perform oxygenic photosynthesis approximately 3 billion years ago, the ancestors of these extant microorganism were among the first to evolve redox signalling pathways to deal with the toxic repercussions of oxygen evolution. As such, characterising redox-sensitive genetic components in photosynthetic bacteria could provide useful knowledge on the genetic adaptation to an aerobic lifestyle, and facilitate the genetic wiring between photosynthetic bacteria with bioelectrochemical systems.</p>
<h1 id="background-and-rationale">Background and Rationale</h1>
<p>Since the discovery of mathematical patterns in the regulation of lactose metabolism in <em>Escherichia coli</em> <span class="citation" data-cites="Jacob1961"></span>, bacteria have emerged as model organisms for biochemists and synthetic biologists aiming to understand and engineer the genetically-encoded regulation that characterise biological systems <span class="citation" data-cites="Andrianantoandro2006"></span>. It is now known how prokaryotes, to survive in a changing world, have evolved complex signal transduction pathways, networks of genes that equip cells with the ability to sense, process, and adapt to various physico-chemical signals <span class="citation" data-cites="Zhulin2003"></span>. Such environmental stimuli - together with their native biochemical sensing circuits - can be repurposed to regulate the expression of synthetic genetic devices <span class="citation" data-cites="Brophy2014"></span>. This has transformed our understanding of cellular physiology and resulted in many biotechnological applications with great potential for the goals of developing a more sustainable bioeconomy <span class="citation" data-cites="Voigt2020"></span>.</p>
<h2 id="tools-to-connect-bioelectrochemistry-and-synthetic-biology">Tools to Connect Bioelectrochemistry and Synthetic Biology</h2>
<p>The majority of such synthetic genetic circuits rely on chemical ligands (e.g. IPTG, antibiotics) as inputs to interface with their native chemosensory signal transduction pathways, such as catabolite repression, chemotaxis and quorum sensing. However, in addition to chemical stimuli, bacteria can also sense physical quantities (e.g. light, temperature), and translate their signals into adaptive genetic responses. By characterising such signal transduction pathways, novel and powerful strategies can be developed to enable user-defined control of biological processes. One example is the field of optogenetics, which uses light as medium to control gene expression. This has proven useful for advanced synthetic biology applications that require precise and reversible spatio-temporal control <span class="citation" data-cites="Lalwani2020"></span>.</p>
<p>Interestingly, evidence also suggests that bacteria, similarly to their eukaryotic descendants, possess the ability to respond to and generate extracellular redox and electrical signals (<span class="citation" data-cites="Green2004"></span>; <span class="citation" data-cites="Martinez-Corral2019"></span>). Genetic components involved in sensing and secreting redox-active compounds have been identified in many bacterial species and have been implicated in a wide array of cellular functions, such as adaptation to toxic redox compounds, intercellular electrical signalling (<span class="citation" data-cites="Prindle201559"></span>; <span class="citation" data-cites="Martinez-Corral2018E8333"></span>), and extracellular electron transfer from cells to soluble compounds and solid electrodes <span class="citation" data-cites="LEASMITH2016247"></span>. From these observations, an interdisciplinary field encompassing synthetic biology and bioelectrochemistry is emerging to understand redox-mediated signal transduction pathways and engineer them to wire the electromagnetic modalities of conventional electronics with the molecular modalities that characterise biological signaling <span class="citation" data-cites="Schofield2020"></span>. This approach aims to elucidate functions of redox-mediated signalling and to provide bioelectrochemical interfaces for direct transfer of information and electricity between electronic and biological devices. For example, an engineered redox-responsive genetic device was recently used to encode binary data into CRISPR arrays of bacterial cells by electrical stimulation <span class="citation" data-cites="Yim2021"></span>.</p>
<h2 id="cyanobacteria-as-chassis-to-investigate-redox-signalling">Cyanobacteria as Chassis to Investigate Redox Signalling</h2>
<p>Unfortunately, synthetic genetic devices that can interface electrochemical and biological signals are limited in their cellular effects to those that are naturally responsive to changes in electron transfer or redox status and, to date, can only operate in model heterotrophic bacteria such as <em>Escherichia coli</em> in aerobic environments (<span class="citation" data-cites="Tschirhart2017"></span>; <span class="citation" data-cites="Liu2016"></span>; <span class="citation" data-cites="Bhokisham2020"></span>). This limited toolbox hinders the investigation of redox signal transduction pathways in other bacteria with different metabolic strategies often employed in electrochemical systems.</p>
<p>An ecologically and biotechnologically important class of bacteria often probed bioelectrochemically, but whose mechanisms of redox signal transduction remain uncharacterised, is formed by cyanobacteria, a phylum of photosynthetic bacteria. These microorganisms are truly unique among biological systems as they combine all of the three bioenergetic processes - fermentation, respiration and oxygenic photosynthesis - in a single (prokaryotic) cell <span class="citation" data-cites="Peschek2011"></span>. Having evolved the ability to perform oxygenic photosynthesis approximately 3 billion years ago <span class="citation" data-cites="Soo2017"></span>, the ancestors of modern cyanobacteria must also have been among the first microorganisms to evolve redox signal transduction pathways to deal with the toxic repercussions of oxygen evolution, such as the oxygen-containing by-products often collectively referred to as reactive oxygen species (ROS) (<span class="citation" data-cites="Sharif2008"></span>; <span class="citation" data-cites="Reniere2018"></span>). In addition, during oxygenic photosynthesis, some of the electrons derived from the photo-oxidation of water are released in the extracellular space and can reduce soluble metal compounds and even solid electrodes <span class="citation" data-cites="Pisciotta2010"></span>. This process, often referred to as exoelectrogenesis, can be exploited to generate electricity in so-called biophotovoltaic devices (BPV): bio-electrochemical systems that harvest electrons generated from photo-oxidation of water by photosynthetic cells at an anode to generate electrical work (<span class="citation" data-cites="Bombelli2011"></span>; <span class="citation" data-cites="McCormick2015"></span>; <span class="citation" data-cites="Wenzel2018"></span>; <span class="citation" data-cites="Zhu2019"></span>). Although the precise mechanisms enabling electrons to flow from cells to electrodes are still unknown, there is evidence suggesting that it might involve the secretion of endogenous redox-mediators (<span class="citation" data-cites="Zhang2018"></span>; <span class="citation" data-cites="Thirumurthy2020"></span>), soluble metabolites that are reduced by cells and reoxidised by electron acceptors such as metal ions or electrodes. Most of the research on this front aims to identify redox-mediators enabling electron transfer from photosynthetic materials to electrochemical devices and to optimise these reactions to maximise the performance of bioelectrochemical devices <span class="citation" data-cites="Wey2019"></span>. However, little is known about the biochemical pathways enabling the flow of redox signals in the inverse directions, that is from electrical to biological devices. Characterising these inward-flowing redox signal transduction pathways in cyanobacteria would be desirable to understand the physiological changes that take place when these bacteria are employed in bioelectrochemial set-ups and to develop synthetic electrogenetic devices that can operate in aerobic photosynthetic hosts.</p>
<h2 id="the-challenges-of-photosynthetic-bacteria">The Challenges of Photosynthetic Bacteria</h2>
<p>Despite these favourable considerations, cyanobacteria also pose challenges for researchers aiming to investigate the genetic components involved in responding to redox signals. In order to be orthogonal to native regulation, these redox signals need not to be photosensitive or interfere with key redox reactions of the photosynthetic electron transport chains. This could result in the generation of unwanted reactive oxygen species and oxidative stress. Due to the lack of well-characterised redox sensitive genetic parts (e.g. transcription factors regulated by redox active compounds), it remains a challenge to identify potential redox conditions to optimise cell-electrode wiring while minimising metabolic damage (e.g. oxidative stress) to the cells. In addition, despite significant progress in the last decade, cyanobacteria offer also more experimental drawbacks compared to other model heterotrophic bacteria, such as the availability of fewer strategies for genetic engineering, a lack of regulatory elements and induction systems and a high genome copy number in many cyanobacterial strains (<span class="citation" data-cites="Huang2010"></span>; <span class="citation" data-cites="Till2020"></span>).</p>
<h2 id="redox-signal-transduction-in-synechocystis-sp.-pcc-6803">Redox Signal Transduction in <em>Synechocystis sp.</em> PCC 6803</h2>
<p>This project aims to address these challenges by investigating and characterising putative genetic parts involved in transducing extracellular redox signals in <em>Synechocystis sp.</em> PCC 6803, a model photosynthetic bacterium commonly used by synthetic biologists and bioelectrochemists. In this strain, many DNA sequences have been identified to be differentially expressed upon various exogenous redox signals (e.g. hydrogen peroxide, redox mediators) (<span class="citation" data-cites="Hihara2003"></span>; <span class="citation" data-cites="RoblesRengel2019"></span>). An interesting one to investigate redox-dependent regulation of gene expression is provided by the largely-uncharacterised prqRA operon. Like the soxRS system (the <em>E. coli</em> redox sensor used in the first and best-characterised electrogenetic device), this is a bicistronic sequence that is involved in the response to oxidative stress, and does not show any light-sensitive regulation. The first gene of the operon, slr0895, encodes the transcription factor PrqR, which has been reported to be highly upregulated upon addition of ROS-generating redox-mediators<span class="citation" data-cites="Khan2016"></span> and when cells are held at positive electrode potentials (Rowden, unpublished). A single point mutation in <em>prqR</em> results in adaptive resistance to the redox-cycling drug methyl viologen (MV) <span class="citation" data-cites="Kirik2003"></span>. MV also binds to PSI and favours production of superoxide anions by Mehler-type reaction and is thus used as a common inducer for oxidative stress <span class="citation" data-cites="Chua1971"></span>. PrqR is a transcriptional repressor for the <em>prqR-prqA</em> operon and controls ROS removal by directly controlling the expression of <em>slr0896</em> and indirectly the iron superoxide dismutase SodB and and the unannotated <em>slr0616</em> <span class="citation" data-cites="Khan2016"></span>. It is unknown whether MV directly derepresses PrqR or whether MV causes an increase in ROS, which indirectly derepress PrqR.</p>
<p>The second gene in the operon, slr0896, encodes the uncharacterised protein PrqA in the Multidrug and Toxin Extrusion (MATE) family with predicted transmembrane, antiporter activity and ion transport. Homologous proteins in other Gram-negative, non-photosynthetic bacteria (e.g.<em>Pseudomonas aeruginosa</em>) utilise an electrochemical potential of H<span class="math inline"><sup>+</sup></span> across the cytoplasmic membrane as the driving force to pump out xenobiotic compounds, including redox-active quinolones and flavin-based antibiotics (<span class="citation" data-cites="Morita1998"></span>; <span class="citation" data-cites="Masuda2000"></span>). Whether this transporter protein, whose expression is regulated by the redox-sensitive transcription factor PrqR, is involved in the transport of redox mediators across cellular membranes remains to be determined.</p>
<figure>
<img src="figs/fintro.png" alt="Electrochemical devices like the voltaic cell shown in blue are systems that can use electrical energy to catalyse redox reactions (allowing electrogenetic control) or, inversely, generating electricity from chemical reactions (exploiting exoelectrogenic bacteria to generate electricity). By imposing a potential energy difference between cathode and anodes, these devices can selectively catalyse oxidation or reduction of redox-mediator compounds (e.g. pyocyanin, methyl viologen etc...) according to their standard redox potential. Identifying redox-mediators that can regulate the activity of genetically-encoded regulators (without interfering with the host photosynthetic redox reactions), allows to investigate how cells can transduce environmental redox signals into biological responses. This project will focus on a specific genetic regulatory system formed by the prqRA operon, which encodes a putative redox-sensitive repressor and an uncharacterised transmembrane protein with predicted transport activity of redox compounds. Abbreviations= PSII: Photosystem II; P680: Photosystem II primary donor pigment; P700: Photosystem I primary donor pigment; PSI: Photosystem II; Q: quinol; b6f: cytochrome b6f; PC: plastocyanin; Fd: ferredoxin; MV: methyl viologen; ROS: reactive-oxygen-species. V: electrode potential." /><figcaption aria-hidden="true">Electrochemical devices like the voltaic cell shown in blue are systems that can use electrical energy to catalyse redox reactions (allowing electrogenetic control) or, inversely, generating electricity from chemical reactions (exploiting exoelectrogenic bacteria to generate electricity). By imposing a potential energy difference between cathode and anodes, these devices can selectively catalyse oxidation or reduction of redox-mediator compounds (e.g. pyocyanin, methyl viologen etc...) according to their standard redox potential. Identifying redox-mediators that can regulate the activity of genetically-encoded regulators (without interfering with the host photosynthetic redox reactions), allows to investigate how cells can transduce environmental redox signals into biological responses. This project will focus on a specific genetic regulatory system formed by the prqRA operon, which encodes a putative redox-sensitive repressor and an uncharacterised transmembrane protein with predicted transport activity of redox compounds. Abbreviations= PSII: Photosystem II; P680: Photosystem II primary donor pigment; P700: Photosystem I primary donor pigment; PSI: Photosystem II; Q: quinol; b6f: cytochrome b6f; PC: plastocyanin; Fd: ferredoxin; MV: methyl viologen; ROS: reactive-oxygen-species. V: electrode potential.</figcaption>
</figure>
<h2 id="aims-and-research-questions">Aims and Research Questions</h2>
<p>In summary, the general scope of the project is to develop a framework to understand and characterise a specific genetic circuit (the prqRA operon) that regulates redox signal transduction in a model cyanobacterium often employed in bioelectrochemical systems. The aim is to understand the bioelectrochemical mechanisms that these photosynthetic bacteria have evolved to respond to exogenous redox stresses. The main hypothesis that underlies this project is that when bacteria are employed in bioelectrochemical systems they may experience both direct and indirect oxidative stresses (e.g. ROS-generating redox mediators, externally-applied oxidative electrode potentials), resulting in upregulation of ROS-quenching adaptive responses (e.g. overexpression of redox-sensitive transcriptional regulators and transmembrane proteins exporting redox-active byproducts). Whereas evidence supporting this hypothesis is available for other bacteria, such as the heterotrophic <em>Pseudomonas</em> <span class="citation" data-cites="Nikel2021"></span> and the chemolitotrophic <em>Geobacter</em> species <span class="citation" data-cites="Levar2017"></span>, its validity and biochemical details still need to be investigated in photosynthetic prokaryotes. Understanding this is of particular biotechnological importance, as it could permit to rationally engineer redox-tolerant cyanobacterial strains with increased exoelectrogenic activity and synthetic electrogenetic plasmids that can operate in oxidative conditions. In order to achieve these goals, this PhD project aims to answer the three big questions listed below. For each of these questions, this study outlines a workflow to solve their more specific and experimentally-answerable subquestions.</p>
<ol>
<li><p>What are the mechanisms and functions of redox transcriptional repression by PrqR ?</p>
<ul>
<li><p>Is it possible to generate markerless mutations in the prqRA operon in <em>Synechocystis</em> cells using Cas12a-mediated gene editing techniques?</p></li>
<li><p>How does the genetic expression from the prqRA operon in mutant strains compare with wild-type cells, in different redox environments (applied external bias potential and after incubation with ROS-generating redox mediators)?</p></li>
<li><p>Do the <span class="math inline"><em>Δ</em></span>prqR mutants show a growth advantage compared to WT, <span class="math inline"><em>Δ</em></span>prqA and <span class="math inline"><em>Δ</em></span>prqRA strains in conditions promoting oxidative stress ?</p></li>
<li><p>Can the phenotypes of the mutants be restored to wild-type when mutant strains are transformed with their respective complementation plasmids?</p></li>
</ul></li>
<li><p>Is it possible to repurpose the redox-sensitive prqRA operon to construct reporter plasmids inducible by externally applied redox signals?</p>
<ul>
<li><p>Does the fluorescence intensity measured in cyanobacteria transformed with genetic construct expressing fluorescent reporter proteins under the control of the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter show a sigmoidal response to the concentration of ROS-generating exogenously-added ligands ?</p></li>
<li><p>Can these redox-active mediators be selectively oxidised or reduced by an external electrode with a midpoint potential orthogonal to those of key photosynthetic reactions?</p></li>
</ul></li>
<li><p>Is it possible to genetically engineer strains of cyanobacteria with increased robustness to oxidative perturbations in order to increase their exoelectrogenic activity ?</p>
<ul>
<li><p>How does the exoelectrogenic activity of <span class="math inline"><em>Δ</em></span>prqR, <span class="math inline"><em>Δ</em></span>prqA and <span class="math inline"><em>Δ</em></span>prqRA mutants compare to wild-type <em>Synechocystis</em> cells?</p></li>
</ul></li>
</ol>
<h1 id="experimental-design">Experimental Design</h1>
<p>To address the questions above this project employs two complementary experimental strategies. First, <em>Synechocystis sp.</em> PCC 6803 cells will be genetically engineered to obtain redox-sensitive mutant strains with deletion in prqR, prqA and the whole prqRA operon, as listed in Table <a href="#table:strains" data-reference-type="ref" data-reference="table:strains">[table:strains]</a>. Then, these mutant strains will be used as hosts to characterise the expression of synthetic reporter plasmids (Figure <a href="#fig:reporters" data-reference-type="ref" data-reference="fig:reporters">2</a>) containing a fluorescent reporter protein cloned downstream of the redox-sensitive <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter, which regulates the expression of the prqRA operon. A theoretical framework to mathematically model the gene expression dynamics from the prqRA operon will be also employed to guide the experimental design and to interpret the results from the experimental characterisation of the reporter plasmids.</p>
<h2 id="rationale-and-strategy-to-generate-mutant-cyanobacteria">Rationale and Strategy to Generate Mutant Cyanobacteria</h2>
<p>All the mutants described in this study are derived from a mother culture of wild-type <em>Synechocystis sp.</em> PCC 6803, which was obtained from a glycerol stock in the Howe lab (David Lea-Smith). These cultures were propagated in standard BG-11 medium in Erlenmeyer flasks shaking at 100 rpm, 30 <span class="math inline"><sup>∘</sup></span>C and 40 <span class="math inline"><em>μ</em></span>mol photons <span class="math inline">$m^{\UTF{2013}2} s^{\UTF{2013}1}$</span> To generate the knock-out mutants, custom plasmids for CRISPR-mediated genome editing have been constructed, as summarised in Figure <a href="#fig:kostra" data-reference-type="ref" data-reference="fig:kostra">1</a>. A CRISPR-based approach has been chosen because it can generate markerless genome modification with a higher efficiency and in a shorter amount of time compared to the classic homologous recombination method <span class="citation" data-cites="Behler2018"></span>. In addition, another PhD student in the Howe lab is developing a similar strategy for CRISPR interference experiments and therefore most of the level 1 CRISPR expression cassettes were already available in the lab. Since the Cas12a protein has been previously shown to be able to work in this host <span class="citation" data-cites="Ungerer2016"></span>, this has been chosen as endonuclease for the genome editing plasmids developed during this study. These plasmids have been assembled with a standardised architecture to enable rapid and modular construction of novel knock-out constructs targeting different genes, which could be helpful for future experiments. Detailed information on the genetic parts included in these plasmids are available in the supplementary information. Briefly, these constructs are derived from the backbone pCAT.011, a mid-copy number replicative plasmid that can propagate in both <em>E. coli</em> and a broad range of cyanobacterial hosts and contain three inserts in the multiple cloning site. Position 1 harbours the coding sequence for Cas12a under the control of constitutive Anderson promoters of increasing strengths. In position 2, under the control of the strongest constitutive promoter (J23119), there is the sequence encoding the guide RNAs with a 23 bp long spacer region targeting the Cas12a to the genomic locus to be cut (at a location of 1819 bases from the 3′ end of the PAM sequence 5′-TTTV-3′). Finally, position 3 is dedicated to the homology repair templates, <span class="math inline">≈</span> 250 bp long DNA sequences homologous to the upstream and downstream regions flanking the cut site and encoding the mutation of interest.</p>
<figure>
<img src="figs/KOstrategy.png" id="fig:kostra" alt="Diagram of the cloning strategy used to construct the knock-out constructs for this study. Regulatory parts like promoters (bent arrows), ribosome binding sites (filled half-circles), terminators (T) and the acceptor backbones have been taken from the CyanoGate and Plant MoClo toolboxes (;). The CRISPR expression cassettes kindly given to me by Joshua Lawrence. The gRNAs and homology repair templates have been cloned during this study from the genome of Synechocystis sp. PCC 6803 via oligo annealing and PCR amplification respectively. The modular nature of this cloning strategy enables to assemble level 0 DNA parts into genetic devices with only 2 one-pot reactions, and is compatible with multipart, combinatorial assembly of DNA sequences with standardised overhangs. " /><figcaption aria-hidden="true">Diagram of the cloning strategy used to construct the knock-out constructs for this study. Regulatory parts like promoters (bent arrows), ribosome binding sites (filled half-circles), terminators (T) and the acceptor backbones have been taken from the CyanoGate and Plant MoClo toolboxes (<span class="citation" data-cites="Vasudevan2019"></span>;<span class="citation" data-cites="Engler2014"></span>). The CRISPR expression cassettes kindly given to me by Joshua Lawrence. The gRNAs and homology repair templates have been cloned during this study from the genome of <em>Synechocystis sp.</em> PCC 6803 via oligo annealing and PCR amplification respectively. The modular nature of this cloning strategy enables to assemble level 0 DNA parts into genetic devices with only 2 one-pot reactions, and is compatible with multipart, combinatorial assembly of DNA sequences with standardised overhangs. </figcaption>
</figure>
<h2 id="constructing-pprqr-reporter-plasmids">Constructing P<sub>prqR</sub> Reporter Plasmids</h2>
<p>To quantify genetic expression from the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter, reporter plasmids have been constructed by cloning the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> sequence upstream of the enhanced Yellow Fluorescent reporter Protein (eYFP). The promoter sequence was identified by bioinformatic analysis (Fig. <a href="#fig:promoteranal" data-reference-type="ref" data-reference="fig:promoteranal">7</a>) and was cloned by annealing and extension of oligonucleotides primers. Different versions and controls for these devices have been constructed. A positive control harbours eYFP upstream of J23119, a very strong, constitutive promoter commonly used as reference in synthetic biology. A negative control was initially devised as the backbone plasmid with no eYFP added. Following discussions with members of my postgraduate panel, it was suggested instead to construct a plasmid containing eYFP and a randomised promoter sequence upstream, to account for leakiness of transcription. A plasmid of this kind (pAST.43) was therefore constructed. Two versions for P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> reporter plasmids have been designed. The first one (pAST.40) is a synthetic version of the prqRA operon, in which the native downstream gene (prqA) was replaced by eYFP. This device will be transformed in <em>Synechocystis</em> cells with the genomic prqR knocked-out (<span class="math inline"><em>Δ</em></span>prqR) and in <em>E. coli</em> to rapidly screen putative PrqR ligands with a plate reader. The second version (pAST.41), contains the eYFP coding sequence downstream of the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter.</p>
<figure>
<img src="figs/reporters.png" id="fig:reporters" alt="Fluorescent reporter devices to quantify genetic expression from P_{prqR}." /><figcaption aria-hidden="true">Fluorescent reporter devices to quantify genetic expression from <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span>.</figcaption>
</figure>
<h2 id="dna-assembly-of-plasmids">DNA Assembly of Plasmids</h2>
<p>All the plasmids generated in this study were assembled using a Golden Gate strategy with MoClo standards developed for synthetic biology in cyanobacteria <span class="citation" data-cites="Vasudevan2019"></span>. This strategy was used because of its high efficiency, the possibility to perform the restriction and ligation step in a one-pot reaction and the availability of standardised overhangs compatible with other cloning systems <span class="citation" data-cites="Weber2011"></span>. All the level 0 and 1 and T plasmids were subcloned in chemically competent <em>E. coli</em> cells, propagated at 37<span class="math inline">∘</span>C in LB media with their respective antibiotics (60 <span class="math inline"><em>μ</em></span>g/mL spectinomycin for level 0 and T, 100 <span class="math inline"><em>μ</em></span>g/mL carbenicillin for level 1). All the level 0 DNA parts were cloned from <em>Synechocystis</em> genomes using primers listed in Table <a href="#table:primers" data-reference-type="ref" data-reference="table:primers">[table:primers]</a> and with standard PCR conditions. Parts shorter than 100 bp (e.g. promoters and gRNAs) were assembled by annealing synthetic oligonucleotides in a PCR thermocycler, as explained in Section <a href="#sec:Annealing" data-reference-type="ref" data-reference="sec:Annealing">5.2</a>. The level T plasmids were transformed in WT <em>Synechocystis sp.</em> cells. Briefly, every DNA assembly reaction was carried out in PCR tubes with 1<span class="math inline"><em>μ</em></span>l of restriction enzyme (BbsI-HF for level 0 and T, BsaI-HF for level T; both from 20,000 units/ml stocks from NEB), 1 <span class="math inline"><em>μ</em></span>l of T4 DNA ligase (from 5u/<span class="math inline"><em>m</em><em>u</em></span>l stock, <a href="https://www.thermofisher.com/order/catalog/product/EL0011#/EL0011">ThermoFisher</a>), 2<span class="math inline"><em>μ</em></span>l ATP (from 100 mM stock), 2<span class="math inline"><em>μ</em></span>l BSA (from 20 ml/ml stock, <a href="https://www.thermofisher.com/order/catalog/product/B14?uk&amp;en#/B14?uk&amp;en">ThermoFisher</a>), 2<span class="math inline"><em>μ</em></span>l buffer G (from 10x stock, <a href="https://www.thermofisher.com/order/catalog/product/BG5?uk&amp;en#/BG5?uk&amp;en">ThermoFisher</a>), 0.75 ng of purified backbone plasmid and a 2:1 insert to backbone ratio. These tubes were then placed in a thermocycler programmed to perform 30 cycles at 37<span class="math inline"><sup>∘</sup></span>C (to activate restriction enzyme) and 16<span class="math inline"><sup>∘</sup></span>C (to activate T4 DNA ligase) of 5 min each. The assembly reaction mixtures were then added with 10 <span class="math inline"><em>μ</em></span>l of 5x KCM to 40 <span class="math inline"><em>μ</em></span>l of chemically competent cells (<a href="https://international.neb.com/products/c2987-neb-5-alpha-competent-e-coli-high-efficiency">NEB 5-alpha</a>) in sterile PCR tubes. Heat-shock was performed in a PCR thermocycler according to the instructions of the competent cell manufacturer. The transformation reaction mixtures were spreaded onto LB plates supplemented with 1.5 % agar, 100mM IPTG (from 1000x stock), 20 mg/ml X-Gal (from 1000x stock) and the appropriate antibiotics. After overnight incubation at 37<span class="math inline"><sup>∘</sup></span>, white colonies were picked from plates and grown overnight in liquid LB medium (with respective antibiotic). After 16-24 hours, cultures were centrifuged and their plasmid DNA extracted with a Miniprep kit (<a href="https://www.thermofisher.com/order/catalog/product/K0503?ICID=cvc-pdna-miniprep-c1t1#/K0503?ICID=cvc-pdna-miniprep-c1t1">ThermoFisher</a>). The concentration and purity of the purified plasmids was measured with a Nanodrop and then sent for Sanger DNA sequencing.</p>
<h2 id="sec:transf">Natural Transformation of <em>Synechocystis sp.</em> PCC 6803</h2>
<p>After having assembled the knock-out and reporter plasmids in <em>E. coli</em> these plasmids were sequenced and transformed into wild-type <em>Synechocystis sp.</em> PCC 6803 cells following the natural transformation protocol described by <span class="citation" data-cites="Pope2020"></span>. Three separate colonies <em>Synechocystis</em> were picked from a BG-11 plate, previously streaked with an axenic culture from a glycerol stock in the lab. These colonies were inoculated in 10 ml of low-phosphate liquid BG-11 medium without glucose, and grown under 40 <span class="math inline"><em>μ</em></span>mol photons <span class="math inline">$m^{\UTF{2013}2} s^{\UTF{2013}1}$</span> of fluorescent white light, shaking at 100 rpm at 31<span class="math inline">∘</span>C in air for <span class="math inline">≈</span> 7 days. Then 1 mL of cells was collected by centrifugation in a 2 ml Eppendorf tube, washed twice in no-phosphate BG-11 and resuspended in 1 ml of no-phosphate BG-11 to give a final OD740 of 1. Cells were collected again by centrifugation and 1 <span class="math inline"><em>μ</em></span>g of plasmid DNA was added to the tube. In the wells of a 12-well microplate, no-phosphate BG-11 was added to a final volume of 100 <span class="math inline"><em>μ</em></span>l. Cells were resuspended and the microplate was sealed with parafilm and left in 40 <span class="math inline"><em>μ</em></span>mol photons <span class="math inline">$m^{\UTF{2013}2} s^{\UTF{2013}1}$</span>) in 30<span class="math inline"><sup>∘</sup></span>C for 6 h, shaking at 100 rpm. After 6 h, 10 <span class="math inline"><em>μ</em></span>l of cell suspension was spotted onto a no-phosphate BG-11 agar plate. The plate was incubated at 30<span class="math inline"><sup>∘</sup></span>C under 40 <span class="math inline"><em>μ</em></span>mol photons <span class="math inline">$m^{\UTF{2013}2} s^{\UTF{2013}1}$</span> of white light for 24 h. This plate is used as a positive control to calculate the maximum number of colonies appearing on plates prior to the addition of negative selection. After 24 h the liquid cultures in the wells of the microplate were centrifuged and re-suspended again in 100 <span class="math inline"><em>μ</em></span>l of no-phosphate BG-11 containing the negative selection marker (spectinomycin) at a final concentration of 60 <span class="math inline"><em>μ</em></span>g/ml. After 24 h, 10 <span class="math inline"><em>μ</em></span>l of this cell suspension was spotted onto a no-phosphate BG-11 agar plate containing spectinomycin) at a final concentration of 60 <span class="math inline"><em>μ</em></span>g/ml. This plate was incubated at 30<span class="math inline"><sup>∘</sup></span>C under 40 <span class="math inline"><em>μ</em></span>mol photons <span class="math inline">$m^{\UTF{2013}2} s^{\UTF{2013}1}$</span> of white light until the appearance of colonies. Cells that have been successfully transformed with plasmids containing spectinomycin-resistant should grow on this plate. By dividing the number of colony forming units on this plate by the number of colonies growing in the positive control plate (no negative selection), it is possible to estimate the transformation efficiency for this experiment.</p>
<h1 id="results">Results</h1>
<h2 id="assembly-of-knock-out-and-reporter-plasmids">Assembly of Knock-Out and Reporter Plasmids</h2>
<p>All the assembled genetic parts have been subcloned in <em>E. coli</em>. After picking and extracting the plasmid content from the white colonies growing in the transformation plates, the DNA sequences have been validated by aligning the Sanger sequencing reads with the <em>in silico</em> designs of the constructs. The tables below summarise these results and show the progress of the cloning strategy performed since the start of this project.</p>
<div id="table:level0">
<table>
<caption>List of level 0 plasmids. Inserts are cut out with restriction enzyme BsaI. Positive selection: spectinomycin. Green: part has been been succesfully assembled (DNA sequencing alignment available in hyperlink). Yellow: part has been assembled but contains undesired mutations. Red: part has not been assembled yet.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Part</strong></th>
<th style="text-align: center;"><strong>Backbone</strong></th>
<th style="text-align: center;"><strong>Insert</strong></th>
<th style="text-align: center;"><strong>Type</strong></th>
<th style="text-align: center;"><strong>DNA Seq</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">pAS0.1</td>
<td style="text-align: center;">pICH41308</td>
<td style="text-align: center;">prqR</td>
<td style="text-align: center;">PrqR coding sequence</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-13_D01_ASAS01ASP1-alignment.pdf">ASAS01ASP1</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">pAS0.2</td>
<td style="text-align: center;">pICH41308</td>
<td style="text-align: center;">prqA</td>
<td style="text-align: center;">PrqA coding sequence</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-10-29_E10_JLPAS02SP1-alignment.pdf">JLPAS02SP1</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">pAS0.5</td>
<td style="text-align: center;">pICH41331</td>
<td style="text-align: center;">prqRA HRT</td>
<td style="text-align: center;">homology repair template (<span class="math inline"><em>Δ</em></span>prqRA)</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-10-29_E10_JLPAS02SP1-alignment.pdf">ASPAS5SP1</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">pAS0.6</td>
<td style="text-align: center;">pICH41295</td>
<td style="text-align: center;"><span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span></td>
<td style="text-align: center;">native promoter + RBS</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-03_G01_ASASO6ASP1-alignment.pdf">ASASO6ASP1</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">pAS0.7</td>
<td style="text-align: center;">pICH42120</td>
<td style="text-align: center;">gRNA-prqR</td>
<td style="text-align: center;">cas12a gRNA</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-07-28_H07_AS7SP1-alignment.pdf">AS7SP1</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">pAS0.8</td>
<td style="text-align: center;">pICH41331</td>
<td style="text-align: center;">prqR HRT</td>
<td style="text-align: center;">homology repair template (<span class="math inline"><em>Δ</em></span>prqR)</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-07-30_E11_ASPAS8SP1-2020-07-30_E11_ASPAS8SP1ab1-alignment.pdf">ASPAS8SP1</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">pAS0.10</td>
<td style="text-align: center;">pICH42120</td>
<td style="text-align: center;">gRNA prqA</td>
<td style="text-align: center;">cas12a gRNA</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-09-07_B02_ASAS010SP1-alignment.pdf">AS10SP1</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">pAS0.11</td>
<td style="text-align: center;">pICH41331</td>
<td style="text-align: center;">prqA HRT</td>
<td style="text-align: center;">homology repair template (<span class="math inline"><em>Δ</em></span>prqA)</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-03_A02_ASASO11ASP1-alignment.pdf">ASASO11ASP1</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">pAS0.12</td>
<td style="text-align: center;">pICH41295</td>
<td style="text-align: center;">PprqA</td>
<td style="text-align: center;">native promoter TSS</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-13_H01_ASAS012ASP1-alignment.pdf">ASAS012ASP1</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">pAS0.13</td>
<td style="text-align: center;">pICH41276</td>
<td style="text-align: center;">TprqR</td>
<td style="text-align: center;">native terminator</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-03_E01_ASASO13ASP1-alignment.pdf">ASASO13ASP1</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">pAS0.14</td>
<td style="text-align: center;">pICH41308</td>
<td style="text-align: center;">mScarletI</td>
<td style="text-align: center;">reporter coding sequence</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L0/2020-11-16_G09_ASASO14BSP1-alignment.pdf">ASASO14BSP1</a></td>
</tr>
</tbody>
</table>
</div>
<p>The majority of the required level 0 parts (which contain basic genetic units such as coding and regulatory sequences) have been successfully assembled. Only the coding sequence for the the PrqR protein needs to be reassembled, as it contains a BbsI recognition site that needs to be domesticated.</p>
<div id="table:level1">
<table>
<caption>List of level 1 plasmids. Inserts are cut out with restriction enzyme BbsI. Positive selection: ampicillin/carbenicillin. Green: part has been been succesfully assembled (DNA sequencing alignment available in hyperlink). Yellow: part has been assembled but contains undesired mutations. Red: part has not been assembled yet.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Part</strong></th>
<th style="text-align: center;"><strong>Backbone</strong></th>
<th style="text-align: center;"><strong>Insert(s)</strong></th>
<th style="text-align: center;"><strong>Type</strong></th>
<th style="text-align: center;"><strong>DNA Seq</strong></th>
<th style="text-align: center;"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">pAS1.1</td>
<td style="text-align: left;">L1P3</td>
<td style="text-align: center;">pAS0.8</td>
<td style="text-align: center;">prqR HRT</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-09-07_E02_ASAS112SP3-2020-09-07_E02_ASAS112SP3ab1-alignment.pdf">AS112SP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.2</td>
<td style="text-align: left;">L1P3</td>
<td style="text-align: center;">pAS0.5</td>
<td style="text-align: center;">prqRA HRT</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-09-07_F02_ASAS12SP3-2020-09-07_F02_ASAS12SP3ab1-alignment.pdf">AS121SP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.3</td>
<td style="text-align: left;">L1P2</td>
<td style="text-align: center;">J23108<span class="math inline"><sub><em>T</em><em>S</em><em>S</em></sub></span>+pAS0.7+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">prqR gRNA1</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-09-07_G02_ASAS131SP3-2020-09-07_G02_ASAS131SP3ab1-alignment.pdf">AS131SP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.4</td>
<td style="text-align: left;">L1P2</td>
<td style="text-align: center;">J23119<span class="math inline"><sub><em>T</em><em>S</em><em>S</em></sub></span>+pAS0.7+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">prqR gRNA2</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-19_F08_ASAS14BSP3-alignment.pdf">AS14BSP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.6</td>
<td style="text-align: left;">L1P3</td>
<td style="text-align: center;">pAS0.11</td>
<td style="text-align: center;">prqA HRT</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-19_B09_ASAS16BSP3-alignment.pdf">AS16BSP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.7</td>
<td style="text-align: left;">L1P1</td>
<td style="text-align: center;">pAS0.6+pAS0.1+pAS0.13</td>
<td style="text-align: center;">prqR</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-19_D09_ASAS17BSP3-alignment.pdf">AS17BSP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.8</td>
<td style="text-align: left;">L1P2</td>
<td style="text-align: center;">J23108<span class="math inline"><sub><em>T</em><em>S</em><em>S</em></sub></span>+pAS0.10+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">prqA gRNA1</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-13_E02_ASAS18BSP4-alignment.pdf">AS18BSP4</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.9</td>
<td style="text-align: left;">L1P2</td>
<td style="text-align: center;">J23119<span class="math inline"><sub><em>T</em><em>S</em><em>S</em></sub></span>+pAS0.10+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">prqA gRNA2</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.10</td>
<td style="text-align: left;">L1P1</td>
<td style="text-align: center;">J23119<span class="math inline"><sub><em>v</em>02</sub></span>+pAS0.1+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">PrqR overexpression</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.11</td>
<td style="text-align: left;">L1P2</td>
<td style="text-align: center;">pAS0.12+eYFP+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>A</em></sub></span> eYFP reporter</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-19_B10_ASAS111BSP3-alignment.pdf">AS111BSP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.13</td>
<td style="text-align: left;">L1P1</td>
<td style="text-align: center;">J23119<span class="math inline"><sub><em>v</em>02</sub></span>+eYFP+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">eYFP +ve control</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-11-19_F10_ASAS113BSP3-alignment.pdf">AS113BSP3</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAS1.14</td>
<td style="text-align: left;">L1P1</td>
<td style="text-align: center;">pAS0.6+eYFP+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span>-eYFP reporter</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-12-01_E06_ASJL114ASP4-alignment.pdf">JL114ASP4</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAS1.15</td>
<td style="text-align: left;">L1P1</td>
<td style="text-align: left;">J23119<span class="math inline"><sub><em>v</em>02</sub></span>+pAS0.14+T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span></td>
<td style="text-align: center;">mScarletI +ve control</td>
<td style="text-align: center;"><a href="https://scaralbi.github.io/assets/dna/L1/2020-12-15_F07_JLJL115ASP4-alignment.pdf">JL115ASP4</a></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
</div>
<p>The majority of the required level 1 constructs (which encode expression cassettes and specify the position of these parts in the final level T vectors) have also been successfully validated by DNA sequencing. One gRNA expression unit (under control of strong J23119 promoter) and the PrqR overexpression construct (which required the correct PrqR coding sequence) still need to be reassembled.</p>
<div id="table:levelT">
<table>
<caption>List of level T plasmids. Do not contain BbsI or BsaI sites. Positive selection: spectinomycin. Green: part has been been succesfully assembled (DNA sequencing alignment available in hyperlink). Yellow: part has been assembled but contains undesired mutations. Red: part has not been assembled yet.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Part</strong></th>
<th style="text-align: center;"><strong>Backbone</strong></th>
<th style="text-align: center;"><strong>Insert(s)</strong></th>
<th style="text-align: center;"><strong>Type</strong></th>
<th style="text-align: center;"><strong>DNA Seq</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">pAST.000</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.017+pAS1.4+pAS1.1</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqR J23105-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.001</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.016+pAS1.4+pAS1.1</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqR J23114-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAST.002</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.015+pAS1.4+pAS1.1</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqR J23117-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.010</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.017+pAS1.3+pAS1.2</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqRA J23105-cas12a</td>
<td style="text-align: left;">T15BASTF</td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAST.011</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.016+pAS1.3+pAS1.2</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqRA J23114-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.012</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.015+pAS1.3+pAS1.2</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqRA J23117-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAST.020</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.017+pAS1.8+pAS1.6</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqA J23105-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.021</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.016+pAS1.8+pAS1.6</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqA J23114-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAST.022</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pJML1.015+pAS1.8+pAS1.6</td>
<td style="text-align: left;"><span class="math inline"><em>Δ</em></span>prqA J23117-cas12a</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.040</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pAS1.7 + pAS1.11</td>
<td style="text-align: left;">prqRA reporter1</td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">pAST.041</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pAS1.14</td>
<td style="text-align: left;">prqRA reporter 2</td>
<td style="text-align: left;"><a href="https://scaralbi.github.io/assets/dna/LT/2020-12-10_A10_AST41DSPTF-2020-12-10_G09_AST41BSPTF-2020-12-10_F09_AST41ASPTF-2020-12-10_H09_AST41CSPTF-alignment.pdf">T41STF</a></td>
</tr>
<tr class="even">
<td style="text-align: left;">pAST.042</td>
<td style="text-align: left;">pCAT.011</td>
<td style="text-align: left;">pAS1.13</td>
<td style="text-align: left;">eYFP +ve control</td>
<td style="text-align: left;"><a href="https://scaralbi.github.io/assets/dna/LT/2020-12-10_B10_AST42ASPTF-alignment.pdf">T42STF</a></td>
</tr>
</tbody>
</table>
</div>
<p>With regards to the knock-out level T plasmids, at least one version for every type of knock-out strain (<span class="math inline"><em>Δ</em></span>prqR, <span class="math inline"><em>Δ</em></span>prqA, <span class="math inline"><em>Δ</em></span>prqRA ) has been assembled. Additional versions of these constructs with cas12a under the control of other promoters still need to be re-assembled. For the reporter plasmids, 2 out of the 3 constructs required have been validated. The remaining plasmid requires the correct version of the prqR coding sequence, which has not been domesticated yet.</p>
<h2 id="sec:trans">Transformation and Selection of Mutants</h2>
<p>After validating the sequences of the level T plasmids, some of those that have been successfully assembled have been transformed in wild-type <em>Synechocystis sp.</em> PCC 6803 cells according to the natural transformation protocol described in Section <a href="#sec:transf" data-reference-type="ref" data-reference="sec:transf">2.4</a>. The backbone plasmid (containing the spectinomycin resistance gene already validated to work in <em>Synechocystis sp.</em> PCC 6803 by <span class="citation" data-cites="Vasudevan2019"></span> have been transformed as positive controls. For the negative control, no plasmid DNA was added. Figure <a href="#fig:transplate" data-reference-type="ref" data-reference="fig:transplate">3</a> shows the outcome of this experiment for three biological replicates of WT <em>Synechocystis sp. PCC</em> transformed with the plasmids labelled on the left and spotted (at different dilutions) on BG11 plates containing specinomycin as selection marker.</p>
<figure>
<img src="figs/trans.png" id="fig:transplate" alt="BG-11 + Spectinomycin plate spotted with cultures of Synechocystis sp. PCC 6803. These results do not prove that the colonies growing on these plates contain the desired genotypes, but only that the transformation protocol and antibiotic selection employed work." /><figcaption aria-hidden="true">BG-11 + Spectinomycin plate spotted with cultures of <em>Synechocystis sp.</em> PCC 6803. These results do not prove that the colonies growing on these plates contain the desired genotypes, but only that the transformation protocol and antibiotic selection employed work.</figcaption>
</figure>
<h2 id="sec:charactreporter">Characterisation of Reporter Constructs</h2>
<p>As shown in Table <a href="#table:levelT" data-reference-type="ref" data-reference="table:levelT">3</a>, most of the sequences of the reporter constructs have been succesfully validated by DNA sequencing. Before transformation in <em>Synechocystis sp.</em> PCC 6803, an experiment was performed to characterise these plasmids in <em>E. coli</em>. Constructs that express in <em>E. coli</em> also usually work in <em>Synechocystis sp. PCC</em> 6803 (although with different dynamic profiles). With this assumption, the reason behind this experiment was to prove that these constructs (that have been genotypically validated) also resulted in the phenotypic emission of a fluorescent signals characteristic of eYFP. Although their DNA sequences were correct, given the stochastic nature of biological systems it seemed cautious not to take this for granted. Using <em>E. coli</em> it was possible to rapidly solve this doubt using a plate reader. This experiment confirms that the constructs encoding eYFP resulted in the emission of a fluorescent signal detected at 527 nm that is characteristic of eYFP (Figure <a href="#fig:platereader" data-reference-type="ref" data-reference="fig:platereader">4</a>).</p>
<figure>
<img src="figs/eYFP_J23119vsPprqR_ecoli_12_2020.png" id="fig:platereader" alt="Negative control: E. coli cells transformed with the same backbone plasmid used to construct the reporter devices but no insert (no eYFP). J23119 (positive control): E. coli cells transformed with pAST.42, encoding eYFP under the control of the strong constitutive J23119 promoter (commonly used as reference in synthetic biology). PprqR: E. coli cells transformed with pAST.41, encoding eYFP downstream of the PprqR promoter native to Synechocystis sp. PCC 6803." /><figcaption aria-hidden="true">Negative control: <em>E. coli</em> cells transformed with the same backbone plasmid used to construct the reporter devices but no insert (no eYFP). J23119 (positive control): <span>E. coli</span> cells transformed with pAST.42, encoding eYFP under the control of the strong constitutive J23119 promoter (commonly used as reference in synthetic biology). PprqR: <em>E. coli</em> cells transformed with pAST.41, encoding eYFP downstream of the PprqR promoter native to <em>Synechocystis sp.</em> PCC 6803.</figcaption>
</figure>
<p>These results also revealed that the steady state fluorescence in cells expressing eYFP under the control of the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter were two orders of magnitude lower than J23119. This is not surprising, given that J23119 is a consensus high-strength promoter and that <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> is not native to <em>E. coli</em>.</p>
<h1 id="discussion-and-future-work">Discussion and Future Work</h1>
<p>The material presented here is the result of the work performed since the start of the PhD project (July 2020), which has mainly focused on molecular biology to generate the genetic constructs needed to answer some of the questions posed in the introduction. The tables of plasmids and their DNA sequencing alignments (Tables <a href="#table:level0" data-reference-type="ref" data-reference="table:level0">1</a>, <a href="#table:level1" data-reference-type="ref" data-reference="table:level1">2</a>, <a href="#table:levelT" data-reference-type="ref" data-reference="table:levelT">3</a>) show that the modular cloning strategy developed and employed in this study was successful to assemble most of the desired genetic constructs needed for future characterisation experiments. The results of the transformation experiments (Section <a href="#sec:trans" data-reference-type="ref" data-reference="sec:trans">3.2</a>) also confirm that the natural transformation protocol is suitable to introduce synthetic plasmids into cyanobacterial cells. As expected, cells transformed with the the plasmids containing the spectinomycin resistance gene aadA as positive selection marker are able to grow in BG-11 plates supplemented with spectinomycin. On the other hand, no colonies appear on this same plate for the negative control condition (no plasmid added). Finally, the results shown in Section <a href="#sec:charactreporter" data-reference-type="ref" data-reference="sec:charactreporter">3.3</a>, show that most of the reporter constructs (needed to quantify expression dynamics from the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter) have also been successfully assembled, validated by DNA sequencing and result in the emission of a fluorescent signal in the yellow channel. The results in Figure <a href="#fig:platereader" data-reference-type="ref" data-reference="fig:platereader">4</a> can be also used to estimate the leaky expression of the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter in <em>E. coli</em>. However, several experiments still need to be performed, and this will be the focus of future work.</p>
<p>In particular, the colonies that appeared after transformation of <em>Synechocystis sp.</em> PCC 6803 cells with the knock-out and reporter plasmids still need to be characterised genotipically and phenotypically in order to validate the cloning strategy presented here. To do so, colonies transformed with the knock-out plasmids will be picked and PCR reactions with primers designed to amplify the genomic region supposed to be edited will be performed. By performing gel electrophoresis and DNA sequencing analyses on these amplicons, it will be possible to assess whether the CRISPR-based plasmid generated the desired deletions in the prqRA loci. For the fluorescent reporter constructs (which are plasmid-based), genotypic validation will be performed by extracting plasmid DNA from the transformed cells and sequence them with primers designed to amplify the reporter constructs. If these analyses reveal that the desired genomic modifications and reporter constructs have been introduced in <em>Synechocystis sp.</em> PCC 6803, the mutants will be the subject of further characterisation experiments (e.g. growth curves) to investigate the phenotypic effects of the introduced mutations. For example, it might be possible that mutants show a significantly reduced growth, which could be due to metabolic burden caused by the synthetic genetic devices or the presence of undesired Cas12a-mediated off-target mutations . To overcome the burden problem, plasmids expressing the Cas12a protein under the control of weaker promoters can be employed to reduce burden (some of these plasmids have already been designed and assembled). To check the presence of off-target mutations, whole genome sequencing can be performed on these mutants. If these experiments will reveal high and potentially-damaging off-target activity, modifications to the design of the plasmid or the transformation protocol can be performed. For example, new plasmids can be assembled to add a counterselectable marker (e.g. sacB) in the backbone. These allow to induce the loss of the plasmid that contain such markers (and the whole CRISPR construct), to minimise the time that these plasmids are inside the cytoplasm of the cells and thus reducing the probability of off-target mutations.</p>
<p>After having validated the presence of the fluorescent reporter constructs inside <em>Synechocystis sp. PCC</em> 6803 cells, these genetically engineered strains will be used to quantify the dynamics of gene expression from the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter when cells grow in a range of different redox conditions (e.g. in the presence of exogenous redox mediators and applied electrode potentials). By quantifying the fluorescence intensity, these experiments will reveal which conditions cause activation of the prqRA promoter and will help to understand the unknown mechanism of redox-regulation by the PrqR repressor. Furthermore, because the PrqA protein has a predicted transporter activity, it would be interesting to quantify the exoelectrogenic activity of the <span class="math inline"><em>Δ</em></span>prqR mutant. In fact - because prqR encodes the transcription factor that represses the expression of prqA - if <span class="math inline"><em>Δ</em></span>prqR mutants show higher exoelectrogenic activity compared to wild-types, this would suggest that PrqA might be involved in exporting redox-mediators. This could help to understand the mechanisms of exoelectrogenesis in photosynthetic bacteria and to engineer highly exoelectrogenic strains to maximise current output in biophotovoltaics.</p>
<p>After having characterised the genotypes and phenotypes of the mutants, an important future experiment is to assemble and generate complementation constructs. These will be assembled by cloning the chromosomal regions that have been deleted for each knock-out mutant into replicative plasmids. These constructs will be used as controls to check if wild-type phenotypes are restored when mutants are transformed with their respective complementation plasmids. If wild-type phenotypes are not restored, this will suggest pleiotropic effects emerging from these mutations, which would be interesting to characterise.</p>
<p>Finally, as a result of the restrictions to laboratory access imposed by the emergence of the COVID-19 pandemic, a bioinformatic “dry" project was devised. This aims to develop a mathematical framework to model the dynamics of gene expression of the prqRA operon in the presence of different putative redox-mediators (more in supplementary information). The theoretical predictions from this model will be useful to guide experimental design and to interpret empirical data from the characterisation of reporter plasmids.</p>
<h1 class="unnumbered" id="acknowledgements">Acknowledgements</h1>
<p>This research was funded by a scholarship from the Biotechnology and Biological Sciences Research Council (BBSRC-DTP). The experimental work was made possible with the supervision of Prof. Christopher Howe. The CyanoGate backbones were a kind gift from Ravi Vasudevan. The cas12a expression units were designed and gifted to me by Joshua Lawrence, to whom I am deeply grateful also for the discussions and cloning sessions we shared. I am thankful to the members of my thesis panel (Dr. Jenny Zhang, Dr. James Locke, Prof. Paul Dupree) for agreeing to be part of this project and for their helpful suggestions. Thanks also to my flatmates, Darius and Cristina, for their company and support during these unprecedented times.</p>
<h1 class="unnumbered" id="references">References</h1>
<div class="center">
<div class="LARGE">
<p><strong>Supplementary Information</strong></p>
</div>
<p><br />
Alberto Scarampi (as2945@cam.ac.uk)<br />
January 2021</p>
</div>
<h1 id="sec:cyanogate">Extended Methods on CyanoGate Assembly</h1>
<p>All the plasmids generated in this study were assembled using Golden Gate, a DNA assembly method that employs Type-IIS restriction enzymes to cut DNA sequences at specific locations and T4 DNA ligase to paste different fragments together. Type-IIS restriction enzymes (BbsI, BsaI) cut DNA at locations outside of their hexameric recognition sequences (e.g. GAAGAC for BBsI) and generate 4-bp long single-stranded overhangs. By designing complementary sets of overhang sequences and assigning standardised overhang to different types DNA parts (e.g. promoters, coding sequences etc...) modular cloning systems (MoClo) have been developed <span class="citation" data-cites="Weber2011"></span>. MoClo systems are available for various synthetic biology chassis and enable a rapid and easy-to-share method to construct modular DNA libraries without proprietary tools and reagents <span class="citation" data-cites="Patron2015"></span>.The MoClo syntax is also seen as the most suitable assembly standard for automated synthesis in DNA Foundries <span class="citation" data-cites="Chambers2016"></span>. For these reasons a MoClo toolbox developed for synthetic biology in cyanobacteria (CyanoGate) was used in this project. This is derived from the MoClo system for plants and microalgae (<span class="citation" data-cites="Engler2014"></span>; <span class="citation" data-cites="Crozet2018"></span>), and has been adopted by various open-source synthetic biology initiatives (<a href="https://www.openplant.org/">OpenPlant</a>, <a href="http://parts.igem.org/Main_Page">iGEM</a>) <span class="citation" data-cites="Vasudevan2019"></span>. In order to make the DNA parts compatible with the CyanoGate toolbox, the genes required in this project were amplified using primers containing part-specific overhangs as listed in Table <a href="#table:overhangs" data-reference-type="ref" data-reference="table:overhangs">4</a>.</p>
<div id="table:overhangs">
<table>
<caption>List of parts and overhang standards for MoClo assembly in cyanobacteria.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Part Type</strong></th>
<th style="text-align: center;"><strong>Description</strong></th>
<th style="text-align: center;"><strong>Backbone</strong></th>
<th style="text-align: center;"><strong>Prefix (5’-3’)</strong></th>
<th style="text-align: center;"><strong>Suffix (5’-3’)</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Prom + 5U</td>
<td style="text-align: center;">promoter + RBS</td>
<td style="text-align: center;">pICH41295</td>
<td style="text-align: center;">GGAG</td>
<td style="text-align: center;">AATG</td>
</tr>
<tr class="even">
<td style="text-align: center;">CDS1</td>
<td style="text-align: center;">coding sequence (with stop codon)</td>
<td style="text-align: center;">pICH41308</td>
<td style="text-align: center;">AATG</td>
<td style="text-align: center;">GCTT</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Cas12 gRNA</td>
<td style="text-align: center;">cas12a direct repeat + spacer</td>
<td style="text-align: center;">pICH42120</td>
<td style="text-align: center;">TAGC</td>
<td style="text-align: center;">TTCG</td>
</tr>
<tr class="even">
<td style="text-align: center;">HDR</td>
<td style="text-align: center;">homology directed repair template</td>
<td style="text-align: center;">pICH41331</td>
<td style="text-align: center;">GGAG</td>
<td style="text-align: center;">TCGC</td>
</tr>
<tr class="odd">
<td style="text-align: center;">3U + Ter</td>
<td style="text-align: center;">3’-UTR / terminator</td>
<td style="text-align: center;">pICH41276</td>
<td style="text-align: center;">GCTT</td>
<td style="text-align: center;">CGCT</td>
</tr>
</tbody>
</table>
</div>
<h2 id="sec:PCR">PCR Amplification to Clone Native Genes into L0 Acceptors</h2>
<p>The native sequences required for this project (homology repair templates, prqR, prqA) were cloned from genomic DNA extracted from <em>Synechocystis sp.</em> PCC 6803. DNA was extracted from a 50 mL culture (grown to stationary phase) using the GeneJET Genomic DNA Purification Kit (Thermo Scientific) according to the manufacturer instructions. PCR reactions to amplify native sequence with primers containing MoClo overhangs (as listed in Table <a href="#table:overhangs" data-reference-type="ref" data-reference="table:overhangs">4</a>) were performed using Phusion High-Fidelity Polymerase (ThermoFisher) and were carried out with the reagents listed in Table <a href="#table:PCRreagents" data-reference-type="ref" data-reference="table:PCRreagents">5</a>.</p>
<div id="table:PCRreagents">
<table>
<caption>List of reagents and their concentrations used for PCR reactions</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Component</strong></th>
<th style="text-align: center;"><strong>[Working]</strong></th>
<th style="text-align: center;"><strong>[Stock]</strong></th>
<th style="text-align: left;"><strong>Volume to add (<span class="math inline"><em>μ</em></span>l per 50 <span class="math inline"><em>μ</em></span>l rxn)</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Autoclaved <span class="math inline"><em>H</em><sub>2</sub><em>O</em></span></td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;">up to 50</td>
</tr>
<tr class="even">
<td style="text-align: center;">Phusion HF Buffer</td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;"><span> 5x</span></td>
<td style="text-align: center;">10</td>
</tr>
<tr class="odd">
<td style="text-align: center;">dNTPs</td>
<td style="text-align: center;">200 nM</td>
<td style="text-align: center;"><span> 10000 nM</span></td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td style="text-align: center;">Genomic DNA</td>
<td style="text-align: center;"><span> 1 ng</span></td>
<td style="text-align: center;"><span> [nanodrop]</span></td>
<td style="text-align: center;">50/[nanodrop]</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Forward Primer</td>
<td style="text-align: center;">0.5 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">25 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td style="text-align: center;">Reverse Primer</td>
<td style="text-align: center;">0.5 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">25 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Phusion<span class="math inline"><sup><em>T</em><em>M</em></sup></span> Polymerase</td>
<td style="text-align: center;">0.02 U/<span class="math inline"><em>μ</em></span>l</td>
<td style="text-align: center;">2 U/<span class="math inline"><em>μ</em></span>l</td>
<td style="text-align: center;">0.5</td>
</tr>
</tbody>
</table>
</div>
<p>PCR reactions were carried out in 0.2 mL tubes in a PCR Thermocycler (Eppendorf). Annealing temperatures (Tm) for each set of primers were calculated with the <a href="https://tmcalculator.neb.com/#!/main">NEB Tm calculator</a> software using the homologous sequences as input. Cycling conditions are listed in Table <a href="#table:cycling" data-reference-type="ref" data-reference="table:cycling">6</a>.</p>
<div id="table:cycling">
<table>
<caption>Parameters for thermal cycling performed to clone parts into level 0 acceptors.</caption>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Cycle Step</strong></th>
<th style="text-align: left;"><strong>Temp (C)</strong></th>
<th style="text-align: left;"><strong>Time (s)</strong></th>
<th style="text-align: left;"><strong>Cycles</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Initial denaturation</td>
<td style="text-align: center;">98</td>
<td style="text-align: center;">30</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">Denaturation</td>
<td style="text-align: center;">98</td>
<td style="text-align: center;">5-10 sec</td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Annealing</td>
<td style="text-align: center;"><span> <strong>Tm</strong></span></td>
<td style="text-align: center;">10-30 sec</td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">Extension</td>
<td style="text-align: center;">72</td>
<td style="text-align: center;">0.015 sec/bp</td>
<td style="text-align: center;">25-35</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Final Extension</td>
<td style="text-align: center;">72</td>
<td style="text-align: center;">5-10 min</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">Hold</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">Hold</td>
<td style="text-align: center;">Hold</td>
</tr>
</tbody>
</table>
</div>
<p>The PCR amplicons were then purified with a PCR Cleanup kit (NEB), digested and ligated in the same tube with acceptor plasmids (2:1 insert:backbone ratio), BbsI and T4 DNA Ligase. Sequences were validated by Sanger sequencing following plasmid extraction from white <em>E. coli</em> colonies transformed with the assembly reactions.</p>
<h2 id="sec:Annealing">Oligo Annealing to Clone Short Parts into L0 Acceptors</h2>
<p>Short DNA sequences (e.g. promoters, gRNA) were constructed by annealing synthetic oligonucleotides as described by <span class="citation" data-cites="Stemmer1995"></span>. For sequences shorter than 50 bp, double-stranded genes were assembled by designing sets of complimentary primers with desired overhangs and annealed by heating at 95<span class="math inline"><sup>∘</sup></span>C and cooling slowly to 4<span class="math inline"><sup>∘</sup></span>C. Longer sequences (promoters) or with secondary structures (e.g. terminators) anneal better by designing overlapping primers with 20 bp complementarity and extending them using dNTPs and a DNA polymerase enzyme (e.g. Phusion<span class="math inline"><sup><em>T</em><em>M</em></sup></span>) as in a PCR reaction.</p>
<figure>
<img src="figs/oligos.png" alt="Strategies for annealing and extension of synthetic oligonucleotides" /><figcaption aria-hidden="true">Strategies for annealing and extension of synthetic oligonucleotides</figcaption>
</figure>
<p>For annealing, complementary oligonucleotides were dissolved at a 1:1 molar ratio in an annealing buffer containing 10 mM Tris, 1 mM EDTA, 50 mM NaCl (adjusted to a pH of 8.0). Table <a href="#table:annealingpar" data-reference-type="ref" data-reference="table:annealingpar">7</a> lists the reagents and their concentrations used for these reactions.</p>
<div id="table:annealingpar">
<table>
<caption>List of reagents for annealing single-stranded oligonucleotides</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Component</strong></th>
<th style="text-align: center;"><strong>[Working]</strong></th>
<th style="text-align: center;"><strong>[Stock]</strong></th>
<th style="text-align: center;"><strong><strong>Volume to add (<span class="math inline"><em>μ</em></span>l per 50 <span class="math inline"><em>μ</em></span>l rxn)</strong></strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Primer F</td>
<td style="text-align: center;">2.5 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">25 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td style="text-align: center;">Primer R</td>
<td style="text-align: center;">2.5 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">25 <span class="math inline"><em>μ</em></span>M</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Annealing buffer</td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;">10x</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td style="text-align: center;">milliQ <span class="math inline"><em>H</em><sub>2</sub>0</span></td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;">1x</td>
<td style="text-align: center;">35</td>
</tr>
</tbody>
</table>
</div>
<p>Oligo extension was performed with the PCR parameters listed in Table <a href="#table:PCRreagents" data-reference-type="ref" data-reference="table:PCRreagents">5</a> and <a href="#table:cycling" data-reference-type="ref" data-reference="table:cycling">6</a> with no template DNA (the primers are the template). The melting temperature was calculated using the 20 bp overlap as input in the <a href="https://tmcalculator.neb.com/#!/main">NEB Tm calculator</a> software.</p>
<h2 id="modular-assembly-of-crispr-knock-out-plasmids">Modular Assembly of CRISPR Knock-Out Plasmids</h2>
<p>The native or synthetic DNA parts assembled as above were used to assemble knock-out plasmids required to generate the <span class="math inline"><em>Δ</em></span>prqR, <span class="math inline"><em>Δ</em></span>prqA, <span class="math inline"><em>Δ</em></span>prqRA mutants. Figure <a href="#fig:KOstrategy" data-reference-type="ref" data-reference="fig:KOstrategy">5</a> illustrates the Golden Gate cloning strategy employed to assemble the level 0 parts into level T knock-out plasmids.</p>
<figure>
<img src="figs/ASgate.png" id="fig:KOstrategy" alt="Outline of the cloning strategy to construct modular knock-out plasmids. White and black squares indicate BbsI and BsaI recognition sites respectively. specR: spectinomycin resistance; bla: ampicillin/carbenicillin resistance; lacZ: beta-galactosidase coding sequence used for blue-white screening upon induction with IPTG and X-Gal." /><figcaption aria-hidden="true">Outline of the cloning strategy to construct modular knock-out plasmids. White and black squares indicate BbsI and BsaI recognition sites respectively. specR: spectinomycin resistance; bla: ampicillin/carbenicillin resistance; lacZ: beta-galactosidase coding sequence used for blue-white screening upon induction with IPTG and X-Gal.</figcaption>
</figure>
<h2 id="level-0-assembly">Level 0 Assembly</h2>
<p>To clone PCR products into MoClo acceptors. Restriction: BbsI; selection: spectinomycin.</p>
<h3 id="homology-repair-templates">Homology Repair Templates</h3>
<figure>
<img src="figs/HRT.png" alt="Cloning Homology Repair Templates into Level 0 Acceptors" /><figcaption aria-hidden="true">Cloning Homology Repair Templates into Level 0 Acceptors</figcaption>
</figure>
<h3 id="coding-sequences">Coding Sequences</h3>
<figure>
<img src="figs/CDS.png" alt="Cloning Coding Sequences into Level 0 Acceptors" /><figcaption aria-hidden="true">Cloning Coding Sequences into Level 0 Acceptors</figcaption>
</figure>
<h3 id="cas12a-grnas">Cas12a gRNAs</h3>
<figure>
<img src="figs/gRNA.png" alt="Cloning Cas12a gRNA via oligo annealing into Level 0 acceptors" /><figcaption aria-hidden="true">Cloning Cas12a gRNA via oligo annealing into Level 0 acceptors</figcaption>
</figure>
<p>All the level 0 plasmids marked with a green dot has been validated by DNA sequencing. The coding sequence for the PrqR repressor contains a BbsI recognition site and still needs to be domesticated. Figure <a href="#fig:L0" data-reference-type="ref" data-reference="fig:L0">6</a> illustrates the library of DNA parts constructed during this project.</p>
<figure>
<img src="figs/l0.png" id="fig:L0" alt="Summary of level 0 library of plasmids assembled until now. Green dots indicate DNA parts validated by DNA sequencing. The pAS0.1 plasmid (yellow circle) contains an undesired mutation that still needs to be domesticated." /><figcaption aria-hidden="true">Summary of level 0 library of plasmids assembled until now. Green dots indicate DNA parts validated by DNA sequencing. The pAS0.1 plasmid (yellow circle) contains an undesired mutation that still needs to be domesticated.</figcaption>
</figure>
<h3 id="level-1-assembly">Level 1 Assembly</h3>
<p>Level 1 assembly reactions consist in cloning the inserts contained in level 0 plasmids into level 1 acceptor backbones. During this assembly reaction, the orientation of the level 0 parts will be inversed (genes will be in the reverse strand). This is not a problem as they will be reinverted in the next level T assembly reactions. The CyanoGate level 1 acceptors are derived from the PlantGate MoClo toolbox. There are up to 7 different level 1 acceptors (L1P1, L1P2, ..., L1P7), which specify the position of level 1 expression units for multigene assembly into the final level T plasmids. For example, if the final plasmid requires two expression cassettes, one upstream of the other, the first unit will need to be cloned in L1P1, whilst the second one in L1P2. After level T assembly, the two cassettes will be found one after the other (separated by a 4 bp scar), in the forward strand in the level T plasmid.</p>
<figure>
<img src="figs/l1.png" alt="Overview of the level 1 plasmids designed and assembled in this project." /><figcaption aria-hidden="true">Overview of the level 1 plasmids designed and assembled in this project.</figcaption>
</figure>
<h3 id="level-t-assembly">Level T Assembly</h3>
<p>Level T assembly consists in cloning multigene constructs from level 1 expression cassettes into level T acceptor backbones. Level T acceptors are shuttle vectors that replicate both in <em>E. coli</em> and in a range of cyanobacteria hosts. This assembly step is performed with BbsI for restriction digestion. Selection depends on the antibiotic resistance gene encoded in the level T backbone. For the pCAT.011 (pSEVA421-T) backbone plasmid used in this study, spectinomycin was used as positive selection. This plasmid is a MoClo-compatible vector derived from pSEVA421 and carries the RK2 replicative origin and has an average copy number of <span class="math inline">≈</span> 10 in <em>Synechocystis</em> <span class="citation" data-cites="Vasudevan2019"></span>.</p>
<h3 id="cloning-level-1-parts-into-level-t-acceptors">Cloning level 1 parts into level T acceptors</h3>
<figure>
<img src="figs/LT.png" alt="Cloning Level 1 into Level T Acceptors. SPT are sequencing primers designed to span the entire insert sequence for Sanger DNA sequencing." /><figcaption aria-hidden="true">Cloning Level 1 into Level T Acceptors. SPT are sequencing primers designed to span the entire insert sequence for Sanger DNA sequencing.</figcaption>
</figure>
<figure>
<img src="figs/lt.png" alt="Overview of level T plasmids designed and assembled in this project" /><figcaption aria-hidden="true">Overview of level T plasmids designed and assembled in this project</figcaption>
</figure>
<h1 id="construction-of-fluorescent-reporter-plasmids">Construction of Fluorescent Reporter Plasmids</h1>
<p>In order to quantify the expression dynamics from native genes, a common approach in synthetic biology is to construct synthetic gene circuits by rewiring the transcription factors and promoters to create novel regulatory topologies <span class="citation" data-cites="Chen2018"></span>. During this project, the native PrqR transcription factor and the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter have been used to construct synthetic gene circuits containing fluorescent reporter proteins regulated by these regulatory sequence. To do so, the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter sequence had to be annotated in the genome of <em>Synechocystis sp.</em> PCC 6803. This was performed by identifying the DNA sequence extending 100 bp upstream of the first A nucleotide in the start codon (ATG) of the prqR gene. As shown in Fig. <a href="#fig:promoteranal" data-reference-type="ref" data-reference="fig:promoteranal">7</a>, this sequence was aligned for comparison with the J23119 promoter sequence (which is a consensus eubacterial promoter commonly used as reference in synthetic biology). Using promoter and RBS prediction softwares (<a href="http://www.bacpp.bioinfoucs.com/home">BacPP</a>, <a href="https://salislab.net/software/">RBS calculator</a>), it was possible to identify certain features in the promoter sequence, such as the ribosome binding sites (RBS in green), the -10 and -35 regions (in yellow and purple, the sequences bound by the RNA polymerase sigma factors). The operator sequences (regions bound by repressor PrqR) were previously annotated by <span class="citation" data-cites="Babykin2003"></span>.</p>
<figure>
<img src="figs/promoters.png" id="fig:promoteranal" alt="Comparison of the P_{prqR} and the consensus J23119 promoter." /><figcaption aria-hidden="true">Comparison of the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> and the consensus J23119 promoter.</figcaption>
</figure>
<p>Cyanobacterial promoters can be categorised into three groups based on their sequences and which sigma factors recognise them. Type I promoters (recognised by <span class="math inline"><em>σ</em></span>70) contain both a −10 element (consensus 5’-TATAAT-3’) and a −35 element (consensus 5’-TTGACA-3’). Type II promoters contain the −10 element but lack a −35 element, instead containing operator sites bound by transcription factors. Type III promoters lack −10 or −35 elements and generally respond to various stresses through binding of type III sigma factors <span class="citation" data-cites="Gordon2018"></span>. Bioinformatic analysis of the <span class="math inline"><em>P</em><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter suggested that this promoter might be a type II promoter. In fact it contains a −10 element but the sequence at the −35 location is quite dissimilar to the -35 consensus (5’-TTGACA-3’), and instead contains two operator sites recognised by its repressor PrqR.</p>
<p>After this analysis, the annotated P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter sequence was assembled from annealing and extension of a pair of oligos (P58 and P59 in Table <a href="#table:primers" data-reference-type="ref" data-reference="table:primers">[table:primers]</a>) with overhangs compatible with the level 0 promoters acceptor. This sequence was cloned upstream of the fluorescent protein eYFP. A positive control was constructed by replacing the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> with the J23119 consensus promoter. Terminators (T<span class="math inline"><sub><em>c</em><em>l</em><em>p</em><em>P</em><em>X</em></sub></span>) were added at the 3’ end. The protein eYFP was chosen because of its fluorescent signal orthogonal to to the endogenous background fluorescence of <em>Synechocystis sp.</em> PCC 6803 (Fig. <a href="#fig:fluo" data-reference-type="ref" data-reference="fig:fluo">8</a>).</p>
<figure>
<img src="figs/fspectracells.png" id="fig:fluo" alt="On the left: A colony of \approx 4 million WT Synechocystis sp. cells visualised under a fluorescence microscope (\lambda_{em} = 667 nm). The red signal is the endogenous fluorescence emitted by photosynthetic components (e.g. chlorophylls). On the right: fluorescence spectra of whole cells overlayed with those of the fluorescent reporters eYFP (yellow) and mScarlet-I." /><figcaption aria-hidden="true">On the left: A colony of <span class="math inline">≈</span> 4 million WT <em>Synechocystis sp.</em> cells visualised under a fluorescence microscope (<span class="math inline"><em>λ</em><sub><em>e</em><em>m</em></sub> = 667<em>n</em><em>m</em></span>). The red signal is the endogenous fluorescence emitted by photosynthetic components (e.g. chlorophylls). On the right: fluorescence spectra of whole cells overlayed with those of the fluorescent reporters eYFP (yellow) and mScarlet-I.</figcaption>
</figure>
<h1 id="mathematical-modelling-of-prqr-dynamics">Mathematical Modelling of PrqR Dynamics</h1>
<p>The synthetic negative feedback network displayed by the prqRA operon was formalised using mathematical modelling in MATLAB R2019b. Regulation at both the transcriptional (repressor-promoter) and translational levels was modelled using a simple system of ordinary differential equations (ODE) commonly used to predict expression kinetics of synthetic gene circuits. The autorepressor was simulated using a two-state model similar to previously described models of the TetR autorepressor (<span class="citation" data-cites="Braun"></span>; <span class="citation" data-cites="Kelly2018"></span>). The aim of this modelling is to predict the fluorescence output from the reporter plasmids constructed in this study depending on the concentration of redox-mediator. In one direction (theory <span class="math inline">→</span> experiments), these results can help to identify suitable ranges of concentration of redox mediators to capture the sigmoidal fluorescence output characteristic of inducible promoters. In the other direction (experiments <span class="math inline">→</span> theory), using this theoretical framework it would be possible to estimate the concentration of ligand effector present in the cell given the fluorescence output quantified from the experiments.</p>
<figure>
<img src="figs/modelling.png" alt="Schematic overview of a simple ODE model to simulate dynamics of prqRA operon. The PrqR repressor is an autorepressor of its own P_{prqR} promoter. It has been assumed that the PrqR repressor behaves as other repressor in the tetR-like family. These transcriptional regulators generally exist in two states (ON or OFF) depending on the presence of ligand molecules. In the ON state, they represses transcription from their cis-regulatory sequences (e.g. prqR and eYFP in this case). Binding with ligand X induces a conformational change in the DNA-binding domain of the repressor, resulting in its inactivation (OFF) and consequent transcription of prqR and eYFP genes." /><figcaption aria-hidden="true">Schematic overview of a simple ODE model to simulate dynamics of prqRA operon. The PrqR repressor is an autorepressor of its own P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter. It has been assumed that the PrqR repressor behaves as other repressor in the tetR-like family. These transcriptional regulators generally exist in two states (ON or OFF) depending on the presence of ligand molecules. In the ON state, they represses transcription from their cis-regulatory sequences (e.g. prqR and eYFP in this case). Binding with ligand X induces a conformational change in the DNA-binding domain of the repressor, resulting in its inactivation (OFF) and consequent transcription of prqR and eYFP genes.</figcaption>
</figure>
<h2 id="modelling-gene-expression">Modelling Gene Expression</h2>
<p>Gene expression dynamics for the PrqR-YFP device is modelled with a simple system of kinetic reactions depending on two states: the concentration of mRNA (r) and protein (R). The underlying biochemical reactions (transcription, translation, degradation) can be expressed as:</p>
<p><span class="math display">$$\begin{gathered}    
    \emptyset \xrightarrow{k_{tx}} r \xrightarrow{d} \emptyset\\
    r \xrightarrow{k_{tz}} r + R\\ 
    R \xrightarrow{D} \emptyset
\end{gathered}$$</span> Where <span class="math inline"><em>Θ</em></span> indicate empty sets (genes, degradation), <span class="math inline"><em>k</em><sub><em>t</em><em>x</em></sub></span>is the apparent transcription rate, d is the mRNA degradation rate, <span class="math inline"><em>k</em><sub><em>t</em><em>z</em></sub></span> is the translation rate and D is the protein degradation rate. These reactions are described by two sets of ODEs:</p>
<p><span class="math display">$$\label{eq:transcription1}
    \frac{\delta r}{\delta t} = k_{tx}r - dr$$</span></p>
<p><span class="math display">$$\frac{\delta R}{\delta t} = k_{tz}r - DR$$</span></p>
<h2 id="repressor-and-ligand-binding">Repressor and Ligand Binding</h2>
<p>The binding between repressor and ligand is modelled as a second-order kinetic reaction. The DNA-bound active repressor, <span class="math inline"><em>R</em><sub><em>o</em><em>n</em></sub></span>, reacts with ligand X to form the inactive form of the repressor-ligand complex, <span class="math inline"><em>R</em><sub><em>o</em><em>f</em><em>f</em></sub></span>: <span class="math display">$$R_{on} + nX \stackrel[k_{r}]{k_{f}}{\leftrightarrows} R_{off}$$</span> where n is the number of ligand binding sites in R, <span class="math inline"><em>k</em><sub><em>f</em></sub></span> and <span class="math inline"><em>k</em><sub><em>r</em></sub></span> are the forward and reverse reaction rates. The rate of appearance of the repressor-ligand complex can then be described by the following ODE</p>
<p><span class="math display">$$\label{eq:ligandbinding}
    \frac{R_{off}}{dt} = k_{f}[R_{on}][X]^{n} - k_{r}[R_{off}]$$</span></p>
<p>If <span class="math inline">[<em>R</em><sub><em>o</em><em>f</em><em>f</em></sub>]</span> is initially zero, the concentrations will shift until the forward reaction rate is equal to the reverse reaction rate, such that the concentrations no longer vary with time. At this equilibrium point, the concentration of the complex won’t vary with time. Therefore equating Equation <a href="#eq:ligandbinding" data-reference-type="ref" data-reference="eq:ligandbinding">[eq:ligandbinding]</a> to 0 and after some algebraic rearrangement, the fractions of bound repressor molecules as a function of ligand concentration are described by the following Hill equation:</p>
<p><span class="math display">$$\label{eq:Hill}
    \Theta = \frac{Bound [PrqR]}{Total [PrqR]} =  \frac{[R_{off}]}{[R_{on}] + [X]^{n}}$$</span></p>
<p>Therefore multiplying the total PrqR concentration, R, with the fraction of unbound molecules (1-<span class="math inline"><em>Θ</em></span>) enables to calculate the concentration of active (DNA-bound) repressors at steady state:</p>
<p><span class="math display">$$\label{eq:Ron}
	R_{on} = R\cdot(1-\Theta) = R\cdot(1 - \frac{[X]^n}{Kd_{x} + [X]^n})$$</span></p>
<p>Where Kd<span class="math inline"><sub><em>x</em></sub></span> is the apparent dissociation constant, defined as the ratio of the dissociation rate constant (<span class="math inline"><em>k</em><sub><em>r</em></sub></span>) and the association rate constant (<span class="math inline"><em>k</em><sub><em>f</em></sub></span>) such that:</p>
<p><span class="math display">$$Kd_{x} = \frac{k_{r}}{k_{f}} = \frac{[R_{on}][X]^n}{[R_{off}]}$$</span></p>
<h2 id="transcription-as-a-function-of-active-repressor">Transcription as a Function of Active Repressor</h2>
<p>Equation <a href="#eq:transcription1" data-reference-type="ref" data-reference="eq:transcription1">[eq:transcription1]</a> describes the rate of transcription from the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter but assumes that the promoter is very strong, such that all promoters were bound. However, in the case of prqR-mediated autorepression, we need to account for the amound of promoter sites bound by the repressor. It is assumed that all the repressors unbound by the ligand will bind to the promoters and vice versa. Using the hill equation (Equation <a href="#eq:Hill" data-reference-type="ref" data-reference="eq:Hill">[eq:Hill]</a>), it is possible to describe the transcription from the P<span class="math inline"><sub><em>p</em><em>r</em><em>q</em><em>R</em></sub></span> promoter as a function of the concentration of active repressors, with the generic equation of the form:</p>
<p><span class="math display">$$\label{eq:transrep2}
    \frac{\delta r}{\delta t} = k_{tx} \cdot (1 - \frac{[R_{on}]^{N}}{K_D + [R_{on}]^{N}}) = \frac{k_{tx}}{1 + \frac{[R_{on}]^N}{K_D}}$$</span></p>
<p>Additionally, even when fully bound, the promoter is still able to transcribe the downstream gene, but at a reduced rate. This is described as leaky transcription. The maximum transcription rate, <span class="math inline"><em>K</em><sub><em>t</em><em>x</em></sub></span> is therefore redefined as <span class="math inline"><em>α</em></span> + <span class="math inline"><em>α</em><sub>0</sub></span> in the presence of no repressors bound to the promoter and <span class="math inline"><em>α</em><sub>0</sub></span> when saturated with repressor R. The change in concentration of mRNA over time as a function of the repressor bound to the promoters can be then rewritten as:</p>
<p><span class="math display">$$\frac{\delta r}{\delta t} = \frac{\alpha}{1 + \frac{[R_{on}]^{N}}{K_D}} + \alpha_{0}$$</span></p>
<h2 id="final-system-of-odes">Final System of ODEs</h2>
<p>By substituting the concentration of active repressor calculated in Equation <a href="#eq:Ron" data-reference-type="ref" data-reference="eq:Ron">[eq:Ron]</a> with <span class="math inline"><em>R</em><sub><em>o</em><em>n</em></sub></span> of Equation <a href="#eq:transrep2" data-reference-type="ref" data-reference="eq:transrep2">[eq:transrep2]</a>, results in the final ODEs describing transcription of r as a function of ligand concentration: <span class="math display">$$\label{eq:transcrtiptionrepress}
    \frac{\delta r}{\delta t} = \frac{\alpha}{1 + \frac{(R\cdot(1 - \frac{[X]^n}{Kd_{x} + [X]^n}))^{N}}{K_D}} + \alpha_{0} - dr$$</span></p>
<p>The translation of r into R is given by Equation <a href="#eq:translationtot" data-reference-type="ref" data-reference="eq:translationtot">[eq:translationtot]</a>, which describes the change in protein concentration, R, over time as a function of the translation and degradation of mRNA,r. <span class="math display">$$\label{eq:translationtot}
    \frac{\delta R}{\delta t} = k_{tz}r - DR$$</span></p>
<h2 id="deterministic-simulation">Deterministic Simulation</h2>
<p>Equations <a href="#eq:transcrtiptionrepress" data-reference-type="ref" data-reference="eq:transcrtiptionrepress">[eq:transcrtiptionrepress]</a> and <a href="#eq:translationtot" data-reference-type="ref" data-reference="eq:translationtot">[eq:translationtot]</a> have been solved numerically using Matlab ode45 solver. The parameters used to run the simulation have been taken from biologically similar systems and are listed in Table <a href="#table:params" data-reference-type="ref" data-reference="table:params">8</a>. Future experimental efforts will also involve the quantification of some of these parameters.</p>
<div id="table:params">
<table>
<caption>List of parameters used utilised to simulate the system of ODEs</caption>
<thead>
<tr class="header">
<th style="text-align: left;"><strong></strong></th>
<th style="text-align: left;"><strong>Description</strong></th>
<th style="text-align: left;"><strong>Value</strong></th>
<th style="text-align: left;"><strong>Unit</strong></th>
<th style="text-align: left;"><strong>Reference</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>α</em></span></td>
<td style="text-align: left;">transcription rate of tetR-GFP gene from Ptet</td>
<td style="text-align: left;">0.085</td>
<td style="text-align: left;">nM <span class="math inline"><em>s</em><sup>−1</sup></span></td>
<td style="text-align: left;"><span class="citation" data-cites="Braun"></span></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>α</em><sub>0</sub></span></td>
<td style="text-align: left;">transcription leak rate of tetR-GFP gene from Ptet</td>
<td style="text-align: left;">0.05</td>
<td style="text-align: left;">nM <span class="math inline"><em>s</em><sup>−1</sup></span></td>
<td style="text-align: left;"><span class="citation" data-cites="Kelly2018"></span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">d</td>
<td style="text-align: left;">mRNA degradation rate</td>
<td style="text-align: left;">4.1 <span class="math inline">⋅</span> 10<span class="math inline"><sup>−3</sup></span></td>
<td style="text-align: left;">s<span class="math inline"><sup>−1</sup></span></td>
<td style="text-align: left;"><span class="citation" data-cites="Kelly2018"></span></td>
</tr>
<tr class="even">
<td style="text-align: left;">D</td>
<td style="text-align: left;">protein degradation rate</td>
<td style="text-align: left;">3.9 <span class="math inline">⋅</span> 10<span class="math inline"><sup>−4</sup></span></td>
<td style="text-align: left;">s<span class="math inline"><sup>−1</sup></span></td>
<td style="text-align: left;"><span class="citation" data-cites="Kelly2018"></span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>k</em><sub><em>t</em><em>z</em></sub></span></td>
<td style="text-align: left;">translation rate from from Ptet</td>
<td style="text-align: left;">0.0154</td>
<td style="text-align: left;"><span class="math inline"><em>s</em><sup>−1</sup></span></td>
<td style="text-align: left;"><span class="citation" data-cites="Kelly2018"></span></td>
</tr>
</tbody>
</table>
</div>
<p>As shown in Figure <a href="#fig:modelgraphs" data-reference-type="ref" data-reference="fig:modelgraphs">9</a> (on the left), the model predicts that the higher the concentration of inducer, the higher the steady-state fluorescence intensity. By plotting the steady state fluorescence as a function of ligand concentration, the model also predicts a sygmoidal dose response to the logarithm of ligand concentration. This is characteristic of a double-negative feedback with cooperativity <span class="citation" data-cites="Kelly2018"></span>. The cooperativity parameter (the Hill coefficient n) determines the steepness of the response and can be fitted with experimental data to estimate the number of repressor monomers binding cooperatively to the promoter. These results have been simulated with parameters for the TetR repressor and tetracycline as the inducer. However, in the future, these parameters will be adapted to the ones for the redox-mediator tested experimentally (e.g. methyl viologen, pyocyanin etc...). The reporter devices will be characterised using serial dilutions of a range of different putative inducers. If the experimental data for a given inducer fit with the theoretical predictions of double-negative feedback, this would suggest that the inducer under study is a potential ligand to PrqR.</p>
<figure>
<img src="figs/simulations.png" id="fig:modelgraphs" alt="Numerical simulations of the system of ODEs describing the dynamics of PrqR regulation. On the left:eYFP fluorescence as a function of time for increasing concentrations of putative inducer X (increasing shades of grey). On the right: steady-state fluorescence response as a function of the logarithm of the inducer concentration. Lines of different colours indicate simulations with different Hill coefficients according to legend in the top left corner." /><figcaption aria-hidden="true">Numerical simulations of the system of ODEs describing the dynamics of PrqR regulation. On the left:eYFP fluorescence as a function of time for increasing concentrations of putative inducer X (increasing shades of grey). On the right: steady-state fluorescence response as a function of the logarithm of the inducer concentration. Lines of different colours indicate simulations with different Hill coefficients according to legend in the top left corner.</figcaption>
</figure>
</body>
</html>
